pkgs/coverage/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.14.0-wip
+
+- Require Dart ^3.6
+- Partial support for workspace packages in `test_wth_coverage`.
+- Deprecate `test_wth_coverage`'s `--package-name` flag, because it doesn't make
+  sense for workspaces.
+- Change the default `--port` to 0, allowing the VM to choose a free port.
```

Comment:
Is there any possibility that a user will be assuming that 8181 is being used by default? If the VM service from the spawned process is something we expect users to want to connect to, this is a breaking change in behavior.

Reply:
I considered that, but it seems pretty unlikely. I'd expect people to be manually specifying the port if they need that sort of thing.